src/cli/main.c

```diff
@@ -80,6 +82,9 @@ main(int argc, char *argv[])
 			del_network = 1;
 			network_name = optarg;
 			break;
+		case 'L':
+			new_network_name = optarg;
```

Comment:
I think here you could just reuse the 'network_name' variable that is already declared

Reply:
fixed!

Reply:
I think i twas initially confused by the if (provcode != NULL) { ... } else if (network_name) { ... } and concerned that re-using the network_name variable would cause that clause to be entered if -k and -L were used together. But I realized that if the first bit evaluates to true then we won't necessarily enter the else if clause. I tested the changes and it seems to work well!